launchable/commands/subset.py

```diff
@@ -187,6 +187,15 @@ def subset(
         )
         sys.exit(1)
 
+    if is_observation and is_output_exclusion_rules:
```

Comment:
Users might receive confusing messages like the following when a session has already been created. However, I believe that this is a rare occurrence, so it should not be a significant issue.
```
- WARNING: Both options --observation and --output-exclusion-rules are set. No output will be generated in this case.
- WARNING: --observation flag was ignored. Observation mode can only be enabled for a test session during its initial creation. Add `--observation` option to the `launchable record session` command instead.
```

Reply:
```
WARNING: `--observation` and `--outpu-exclusion-rules` are set. No output will be generated"
```

ref: https://github.com/launchableinc/cli/pull/613

Reply:
And for now, let's allow two warning messages will be displayed and see how it goes

Reply:
Sure, here is the log generated from running the CLI command that you'd like to review.

```
$ /Users/ono-max/.local/share/virtualenvs/cli-o7XEfK99/bin/launchable subset --target 70% --session builds/hogehoge/test_sessions/48 --no-build --rest foo.txt --output-exclusion-rules --observation maven src/test/java

WARNING: Both options --observation and --output-exclusion-rules are set. No output will be generated in this case.
WARNING: `--session` and `--no-build` are set.
Using --session option value (builds/hogehoge/test_sessions/48) and ignoring `--no-build` option
WARNING: --observation flag was ignored. Observation mode can only be enabled for a test session during its initial creation. Add `--observation` option to the `launchable record session` command instead.
```

Reply:
Sorry, this is my suggestion of warning message https://github.com/launchableinc/cli/pull/618#discussion_r1306999738

Reply:
[64751ae](https://github.com/launchableinc/cli/pull/618/commits/64751aee006630f7a13e8fcc2f6f292dd92511fe)